roles/network_plugin/kube-ovn/templates/cni-kube-ovn-crd.yml.j2

```diff
@@ -319,3 +319,81 @@ spec:
       - vpc
```

Comment:
Are those kube-ovn templates changes intended? These changes are from following commit: https://github.com/kubeovn/kube-ovn/commit/1152744e9f47626310601a4c63a7af61e3fe6bf6

It wasn't applied on 1.16 release branch (only on master), so both yamls and code changes are missing in v1.6.1. Otherwise LGTM.

Reply:
Thanks for the review, let me check :)

Reply:
Thanks @maciaszczykm